i18n.js

```diff
@@ -1017,6 +1024,9 @@ module.exports = function() {
     } else {
       // No object notation, just return an accessor that performs array lookup.
       return function() {
+        if (overrides && overrides[locale] && overrides[locale][singular]) {
+          return overrides[locale][singular];
+        }
```

Comment:
LGTM, nice and simple.

This code is insane though.  "delayed access to translations nested inside objects..."?!

Reply:
Completely agree. I cringed the whole time I was writing this :)

I think we'll have the delayed access problem as long as they're dynamically defined. Tally one more risk to mediate with tooling...